pkg/controller/account/ec2.go

```diff
@@ -257,3 +322,277 @@ func ListEC2InstanceStatus(reqLogger logr.Logger, client awsclient.Client) (*ec2
 
 	return result, nil
 }
+
+// getDesiredVCPUValue retrieves the desired quota information from the operator configmap and converts it to a float64
+func (r *ReconcileAccount) getDesiredVCPUValue(reqLogger logr.Logger) (float64, error) {
+	var err error
+	var vCPUQuota float64
+
+	configMap, err := controllerutils.GetOperatorConfigMap(r.Client)
+	v, ok := configMap.Data["quota.vcpu"]
+	if !ok {
+		err = awsv1alpha1.ErrInvalidConfigMap
+	}
+	if err != nil {
+		reqLogger.Error(err, "failed getting desired vCPU quota from configmap data")
+		return vCPUQuota, err
+	}
+
+	vCPUQuota, err = strconv.ParseFloat(v, 64)
+	if err != nil {
+		reqLogger.Error(err, "failed converting vCPU quota from configmap string to float64")
+		return vCPUQuota, err
+	}
+
+	return vCPUQuota, nil
+}
+
+// getVCPUQUota returns the current set vCPU quota for the region
+func getVCPUQuota(client awsclient.Client, desiredQuota float64) (bool, error) {
```

Comment:
If we want to leave the name here as `getVCPUQuota` I think we should be returning an int (or whatever the value that's returned is.  Otherwise I think we should call this something like `validateQuotaIsWithinLimits` or something along those lines.

Reply:
Good call.  This is an old name that needs to be updated.  Will do this tomorrow.

Reply:
Renamed to `vCPUQuotaNeedsIncrease`, as it will return "True" if it does need an increase, to better match the behavior of the function.